NPE if element.definition is empty but contains extensions

If you have a profile which adds extensions to .definition, but doesn't overwrite the actual definition string a NPE is thrown:

```
Cannot invoke "String.startsWith(String)" because the return value of "org.hl7.fhir.r5.model.ElementDefinition.getDefinition()" is null
	at org.hl7.fhir.igtools.publisher.Publisher.generateSnapshot(Publisher.java:7388)
	at org.hl7.fhir.igtools.publisher.Publisher.generateSnapshots(Publisher.java:7298)
	... 4 more
Caused by: java.lang.NullPointerException: Cannot invoke "String.startsWith(String)" because the return value of "org.hl7.fhir.r5.model.ElementDefinition.getDefinition()" is null
```

Code causing the issue:
```java
      if (derived.hasDefinitionElement()) {
        if (derived.getDefinition().startsWith("..."))
```


Profile from wich i derived: https://simplifier.net/medizininformatikinitiative-modulonkologie/mii_pr_onko_diagnose_primaertumor:

```json
"_definition": {
                    "extension":  [
                        {
                            "url": "http://hl7.org/fhir/StructureDefinition/translation",
                            "extension":  [
                                {
                                    "url": "lang",
                                    "valueCode": "de-DE"
                                },
                                {
                                    "url": "content",
                                    "valueString": "Morphologie des Primärtumors nach ICD-O-3 nach 6.3 oBDS"
                                }
                            ]
                        }
                    ]
                }
```

--- a/org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/conformance/profile/ProfileUtilities.java
+++ b/org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/conformance/profile/ProfileUtilities.java
@@ -2656,7 +2656,7 @@ public class ProfileUtilities {
       }
 
       if (derived.hasDefinitionElement()) {
-        if (derived.getDefinition().startsWith("..."))
+        if (derived.getDefinition() != null && derived.getDefinition().startsWith("..."))
           base.setDefinition(Utilities.appendDerivedTextToBase(base.getDefinition(), derived.getDefinition()));
         else if (!Base.compareDeep(derived.getDefinitionElement(), base.getDefinitionElement(), false)) {
           base.setDefinitionElement(derived.getDefinitionElement().copy());

